### PR TITLE
Removed remaining "reference-style links" Markup tagging

### PR DESCRIPTION
**Pull request by mikehoffms**
This PR eliminates all remaining "reference-style links" tagging in the repo, by full-text search on "][" and "]: ".  This PR is solely to change Markdown tagging, including moving part 2 of each link up with part 1, retagged as basic link Markdown.  This PR slightly updates some of the HTML "todo" comments, but analysis of what content changes should be made are out of scope for this PR.

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/javascript/source-maps.md [hunk 1]
```diff
@@ -31,26 +31,35 @@ Keep your client-side code readable and debuggable even after you combine, minif
 *   Only use pre-processors capable of producing Source Maps.
 *   Verify that your web server is able to serve Source Maps.
 
-<!--todo: add link to preprocessors capable of producing Source Maps when section is available -->
-<!--[]: /web/tools/setup/setup-preprocessors?#supported_preprocessors ""  -->
+<!--
+no longer in original file:
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/javascript/source-maps.md:35]: I'm not sure I understand what you mean by "no longer in original file". Can you explain what this line is meant for?

File: microsoft-edge/devtools-guide-chromium/security/index.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ If you have not set up HTTPS on your server, [Let's Encrypt](https://letsencrypt
 
 ### Mixed content
 
-**Mixed content** means that the main origin of a page is secure, but the page requested resources from non-secure origins.  Mixed content pages are only partially protected because the HTTP content is accessible to sniffers and vulnerable to man-in-the-middle attacks.
+**Mixed content**<!--[mixed content](/web/fundamentals/security/prevent-mixed-content/what-is-mixed-content)--> means that the main origin of a page is secure, but the page requested resources from non-secure origins.  Mixed content pages are only partially protected because the HTTP content is accessible to sniffers and vulnerable to man-in-the-middle attacks.
```
Comment (captainbrosset) [inline microsoft-edge/devtools-guide-chromium/security/index.md:66]: I see you've done this a few times in this PR: removed a commented-out reference link from the bottom of the page, and added it as a commented-out inline link next to the corresponding word.

I don't know if the same concept applies here, but when I was doing development, we'd always delete commented-out code. Code that doesn't run is extremely unlikely to ever be used again, and keeping sections of it commented-out doesn't help anybody.
I tend to think that the same thing applies here. The article is likely to evolve over time and the link will remain commented-out.

I would suggest to either use it here (maybe it is useful for users to go and find out more about mixed content), or delete the comment entirely.
What do you think?

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/security/index.md:66]: Note that I understand that this PR is already a good cleanup of the remaining ref links. So it is 100% ok to not answer the above question in the scope of this PR. Land it as is, and then figure out what we want to do with the commented-out links.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/memory-problems/index.md [hunk 3]
```diff
@@ -246,13 +246,12 @@ After you have identified the problem, you are able to use an **Allocation instr
...
-
 > [!NOTE]
 > Portions of this page are modifications based on work created and [shared by Google](https://developers.google.com/terms/site-policies) and used according to terms described in the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
 > The original page is found [here](https://developers.google.com/web/tools/chrome-devtools/memory-problems/index) and is authored by [Kayce Basques](https://developers.google.com/web/resources/contributors#kayce-basques) (Technical Writer, Chrome DevTools \& Lighthouse).
 
 [![Creative Commons License](https://i.creativecommons.org/l/by/4.0/88x31.png)](https://creativecommons.org/licenses/by/4.0)
 This work is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
+
+<!-- [recording](/profile/evaluate-performance/timeline-tool#make-a-recording) -->
+<!-- [hngd](https://jsfiddle.net/kaycebasques/tmtbw8ef/) -->
```
[the remark this answers is not in the document]
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/memory-problems/index.md:257]: Keeping these commented-out links.  In a later PR, compare the original .md file in the other repo, Find these URLs there, analyze, possibly add a todo comment here, or link to the destinations if available at that time and uncomment the link.

**Review by captainbrosset — APPROVED** [review]
Approved based on offline conversations. We're keeping the commented-out links for now.

**Pull request merged**